src/parsers/power_system_table_data.jl

```diff
@@ -1001,15 +1062,8 @@ function make_thermal_generator(data::PowerSystemTableData, gen, cost_colnames,
     fuel = parse_enum_mapping(ThermalFuels, gen.fuel)
```

Comment:
GitHub doesn't let me leave a comment on the correct line, but a few lines up, at
```function make_thermal_generator(data::PowerSystemTableData, gen, cost_colnames, bus)```,
do we want to add the type annotation `cost_colnames::_CostPointColumns` to make sure this only gets called with cost curve-type data?